src/pages/StartPage.tsx

```diff
@@ -0,0 +1,16 @@
+import { Button, Flex, VStack } from '@chakra-ui/react';
+import  Logo from '../components/Logo';
+
+export default function StartPage(){
+    return (
+        <Flex backgroundImage='linear-gradient(to bottom right, green, yellow)' width='100%' height='calc(100vh)' alignItems='center' textAlign='center' justifyContent='center'>
+            <Flex alignItems='center'>
+                <VStack>
+                    <Logo size={["64px", "100px" ]} />
+                    <Button color='white' padding={8} fontSize={["large", '4xl']} colorScheme='whiteAlpha'>CREATE A NEW GAME</Button>
```

Comment:
I suggest making the buttons and text smaller. I think they are a little too close to the size of the logo, and they might be a little too large on mobile.